modules/controller_pool/controller-standby.tpl

```diff
@@ -39,4 +39,4 @@ enable_docker && \
 install_kube_tools && \
 sleep 180 ; \
 backoff_count=`echo $((5 + RANDOM % 100))` ; \
-sleep $backoff_count
+sleep $backoff_count # Shouldn't there be a kubeadm join command somewhere? Looks like we just install tools and do nothing else
```

Comment:
@displague How is this even working? There's no kubeadm call to join the cluster as a controller node

Reply:
This is something I've wanted to take a closer look at and have wanted to refactor for a while:

https://github.com/equinix/terraform-metal-multiarch-k8s/blob/52e29e8eb1dc51f6dd48708816d10b058cacbb16/modules/controller_pool/main.tf#L73-L82

https://github.com/equinix/terraform-metal-multiarch-k8s/blob/main/modules/controller_pool/assets/key_wait_transfer.sh

@rawkode and I have wanted to move the key creation to pre-node creation so that the keys are part of the configuration, not trapped in what should be ephemeral nodes (requiring SSH to move them around).

An example of how this can be done in Pulumi (whose resources are derived from Terraform resources, in this case) is here: https://github.com/equinix-labs/kubernetes-cloud-init/blob/main/src/kubernetes/pki.ts

Reply:
Gotcha - because of how fundamental this is to the control plane pool, it's probably best done in a separate PR